internal/graph/dag.go

```diff
@@ -39,9 +41,10 @@ func (g *DAG[VertexT]) AddEdge(from, to VertexT) error {
 		return ErrAcyclicConstraintViolation
```

Comment:
is it intended that adding duplicate edges returns normally?

Reply:
yeah that should be fixed

Reply:
ok yeah